bulk/eng_bulk_load-AssetRecord.csv

```diff
@@ -171,7 +171,8 @@ CGCON-GLDRCP-00380,OL000355,Sensor,1,Mobile Asset Controller for Coastal Glider,
 CGCON-GLDRCP-00387,OL000356,Sensor,1,Mobile Asset Controller for Coastal Glider,GL387 Controller,Teledyne Webb,G2 Slocum Gliders,387,,,,,Woods Hole Oceanographic Institution
 CGCON-GLDRCP-00388,OL000357,Sensor,1,Mobile Asset Controller for Coastal Glider,GL388 Controller,Teledyne Webb,G2 Slocum Gliders,388,,,,,Woods Hole Oceanographic Institution
 CGCON-GLDRCP-00389,OL000358,Sensor,1,Mobile Asset Controller for Coastal Glider,GL389 Controller,Teledyne Webb,G2 Slocum Gliders,389,,,,,Woods Hole Oceanographic Institution
-CGCON-GLDRCP-00564,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL564 Controller,Teledyne Webb,G2 Slocum Gliders,564,,,,Was Global; now Coastal,Woods Hole Oceanographic Institution
+CGCON-GLDRCP-00564,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL564 Controller,Teledyne Webb,G2 Slocum Gliders,564,,,,,Woods Hole Oceanographic Institution
```

Comment:
Converted from Global unit to Coastal.  Was deployed as a global and as a coastal, so we need two entries.